recipes/aws-noaa-cdr-ndvi/meta.yaml

```diff
@@ -0,0 +1,22 @@
+title: 'AWS NOAA Terrestrial CDR NDVI'
+description: 'Analysis-ready datasets derived from AWS NOAA Terrestrial CDR NDVI NetCDF'
```

Comment:
I think it may be useful to copy over the description from the CDR page from this dataset, like https://github.com/developmentseed/staged-recipes/blob/bf309120ef93003c7e4205ce6e2b6543fe40243c/recipes/aws-noaa-outgoing-longwave-radiation-monthly/meta.yaml#L2

this way I think the original description shows up on the data page? not really sure how the meta.yaml gets served to browsers and end users but I think it does somehow.

Reply:
Good plan. I had followed the description examples I had seen in earlier meta.yaml files, but you're right it cannot hurt to provide the original description, especially if that gets served to browsers/end users. I'll revise.